stacktracer/shared/src/main/scala/scalaz/zio/internal/stacktracer/Tracer.scala

```diff
@@ -0,0 +1,30 @@
+package scalaz.zio.internal.stacktracer
+
+import java.util.concurrent.ConcurrentHashMap
+
+abstract class Tracer extends Serializable {
+  def traceLocation(lambda: AnyRef): ZTraceElement
+}
+
+object Tracer {
+  def globallyCached(tracer: Tracer): Tracer = cached(globalMutableSharedTracerCache)(tracer)
```

Comment:
I'm not sure about the utility of this method. It's used only once in `PlatformLive` initialization, and it's not clear when you'd want several tracers that share a single global cache.

Reply:
I'd say you _always_ want multiple tracers to share a global cache. e.g. if you're running tests in parallel with multiple RTS's if you're using a global cache you can go through the cache warm up for the common code just once, instead of all over again for every test. Since the extracted information is final and determined after class loading, the results from tracing cannot change - as such it nearly never really makes sense to NOT use a global cache

Reply:
All righty! I guess the _multiple RTSes_ use-case wasn't clear to me.